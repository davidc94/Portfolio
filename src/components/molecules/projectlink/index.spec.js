import React from "react"
import renderer from "react-test-renderer"
import ProjectLink from "./index";

describe("ProjectLink", () => {
  it("renders as expected with portfolio", () => {
    expect(renderer.create(<ProjectLink project='portfolio'/>).toJSON()).toMatchSnapshot();
  })
  it("renders as expected with lightscamerahome", () => {
    expect(renderer.create(<ProjectLink project='lightscamerahome'/>).toJSON()).toMatchSnapshot();
  })
  it("renders as expected with moneymanager", () => {
    expect(renderer.create(<ProjectLink project='moneymanager'/>).toJSON()).toMatchSnapshot();
  })
})