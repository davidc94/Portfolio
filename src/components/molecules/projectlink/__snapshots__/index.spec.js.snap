// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectLink renders as expected with lightscamerahome 1`] = `
<a
  href="https://github.com/davidc94/LightsCameraHome"
  style={
    Object {
      "textDecoration": "none",
    }
  }
  target="_blank"
>
  <li
    className="sc-bdVaJa bFKYNT"
  />
</a>
`;

exports[`ProjectLink renders as expected with moneymanager 1`] = `
<a
  href="https://github.com/davidc94/money-manager-poc"
  style={
    Object {
      "textDecoration": "none",
    }
  }
  target="_blank"
>
  <li
    className="sc-bdVaJa bFKYNT"
  />
</a>
`;

exports[`ProjectLink renders as expected with portfolio 1`] = `
<a
  href="https://github.com/davidc94/Portfolio"
  style={
    Object {
      "textDecoration": "none",
    }
  }
  target="_blank"
>
  <li
    className="sc-bdVaJa bFKYNT"
  />
</a>
`;
